package.json

```diff
@@ -339,7 +339,7 @@
     "url-search-params-polyfill": "^8.1.1",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#9e40f2e7eb92b5fddb0ce3b2fb809305abbadf3c",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#",
```

Comment:
The sha is getting removed, please make sure it's present and correct

Reply:
@alexandec , where can I find that? It was removed automatically when I ran `yarn update:schema`.

Reply:
This should come after the `#`:
`e7887afae38610e54dcc9c931dd990682a21e4d7`